src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -0,0 +1,206 @@
+﻿using Xunit;
+using Xunit.Abstractions;
+using System.IO;
+using System.Xml.Schema;
+
+namespace System.Xml.Tests
+{
+    public class TC_SchemaSet_AnyAttribute : TC_SchemaSetBase
+    {
+        private ITestOutputHelper _output;
+
+        public TC_SchemaSet_AnyAttribute(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        public bool bWarningCallback;
```

Comment:
In this case general correct code style. If you want you can also move the others up, but you don't need to.